bait-files/src/main/webapp/SubmissionPage.html

```diff
@@ -0,0 +1,50 @@
+<html>
```

Comment:
General comment on file naming... it's not Googl-specific or an absolute rule, but when naming files of all types in web projects (html, js, css, jpeg, png, etc.), it's a pretty common convention to avoid capital letters and to separate the words in the name with hyphens or underscores (hyphens being more common).  I'd suggest changing these file names to follow that convention:
submission-page.css
submission-page.html

Or you could even use:
submission.css
submission.html
Since the "html" extension pretty much implies that we're dealing with a page =)

Reply:
changed file names